src/styles/blocks/snake.scss

```
@@ -0,0 +1,29 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(3, minmax($size-cell, 1fr));
+
+  @media(max-width: 899px) {
+    grid-template-columns: repeat(2, minmax($size-cell, 1fr));
+  }
+
+  @media(max-width: 599px) {
+    grid-template-columns: 1fr;
+  }
+
+  &__box {
+    display: flex;
```

```suggestion
    display: grid;
```
You could use display: grid for this purpose as well